netmiko/dell/dell_sonic_ssh.py

```diff
@@ -0,0 +1,54 @@
+"""Netimko driver for Dell EMC PowerSwitch platforms running Dell EMC Open Networking Powered by SONiC"""
+import time
+from netmiko.cisco_base_connection import CiscoSSHConnection
+from netmiko import log
+
+class DellSonicSSH(CiscoSSHConnection):
+    """driver for Dell EMC PowerSwitch platforms running Dell EMC Open Networking Powered by SONiC"""
+
+    def session_preparation(self):
+        """Prepare the session after the connection has been established."""
+        self._test_channel_read(pattern=r"[>$#]")                
+        # Clear the read buffer
+        time.sleep(0.3 * self.global_delay_factor)
+        self.clear_buffer()
+        self._enter_shell()
```

Comment:
Should explicitly call `disable_paging()` in `session_preparation` (as I/others are expecting to see it there). This is instead of embedding it in `_enter_shell()`